### PR TITLE
Check for TXs to Hotel in getDecodedTransactions

### PR DESCRIPTION
**Pull request by adklempner**

### CODE REVIEW

**Comment by adklempner** [comment]
I was using the demo-app to manually test it, but I will add unit tests.

File: test/HotelManager.js [hunk 1]
```diff
@@ -138,20 +138,6 @@ describe('HotelManager', function() {
       const newName = 'Awesome WTHotel';
       const newDescription = 'Awesome Winding Tree Hotel';
 
-      let callBacks = {};
```
Comment (AugustoL) [inline test/HotelManager.js]: this callbacks should be removed, but this changes are unrelated with this PR, @adklempner please remove this changes and lets do it in another PR.
